inno_setup.iss

```diff
@@ -379,6 +408,10 @@ begin
   if Length(smrom) = 0 then
     SMRomFilePage:= AddRomPage('Super Metroid (JU).sfc');
 
+  dkc3rom := CheckRom('Donkey Kong Country 3 - Dixie Kong's Double Trouble! (USA) (En,Fr).sfc', '120abf304f0c40fe059f6a192ed4f947');
+  if Length(dkc3rom) = 0 then
+    DKC3RomFilePage:= AddRomPage('Donkey Kong Country 3 - Dixie Kong's Double Trouble! (USA) (En,Fr).sfc');
```

Comment:
Need to Either wrap this in double quotes or add an additional single quote after `Dixie Kong` because it terminates the string early. 

For example change to:
`DKC3RomFilePage:= AddRomPage('Donkey Kong Country 3 - Dixie Kong''s Double Trouble! (USA) (En,Fr).sfc');`